src/app/app-routing.ts

```diff
@@ -147,6 +147,20 @@ export const routes: Routes = [
     pathMatch: 'full',
     redirectTo: 'donate/:campaignId',
   },
+  {
+    path: `${myRegularGivingPath}/:mandateId/thanks`,
```

Comment:
Wondering if this might be slightly quicker for devs to follow if we used a query param instead of an extra path piece? But I don't feel strongly, this is also good if you think 2 routes is clearer.

Reply:
I think I'd rather keep it as two routes, I think it looks neater without query params, and using a query param more invites users to edit the URL manually. These are really two different pages that just happen to be very similar and use the same data from the backend. I partly thinking it might be better if they were separate components, since there's several bits varied, but I won't change that now.